week_12/진명재/14400.java

```diff
@@ -0,0 +1,36 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.StringTokenizer;
+
+public class 14400 {
+    public static void main(String[] args) throws Exception {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+        int N = Integer.parseInt(br.readLine());
+
+        int[] x = new int[N];
+        int[] y = new int[N];
+
+        int m = N / 2;
```

Comment:
```suggestion
        int m = N >> 1;
```
이번에 배운 비트마스킹 이렇게 한번이면 안써도 되긴함